forms-flow-admin/src/components/dashboard/dashboard.tsx

```diff
@@ -324,6 +282,19 @@ export const InsightDashboard = React.memo((props: any) => {
               noDataIndication={noData}
               data-testid="admin-dashboard-table"
             />
+            <table className="table">
+              <tfoot>
+              <TableFooter
```

Comment:
now it is okey, but add  comment like need to change  bootstrap table to our custom table else two tables will come instead of one @Bonymol-aot

Reply:
please add package-lock too and ensure the node version is 16.18.0 @Bonymol-aot

Reply:
Resolved